extra/docs_landing.html

```diff
@@ -48,335 +48,195 @@
-      </li>
-      <li>
-        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/elastic.ec-azure-pp?tab=Overview">Azure Marketplace</a>
-      </li>
-      <li>
-        <a href="https://console.cloud.google.com/marketplace/product/endpoints/elasticsearch-service.gcpmarketplace.elastic.co">GCP Marketplace</a>
-      </li>
-      <li>
-        <a href="https://cloud.elastic.co/pricing">Review pricing using our calculator</a>
-      </li>
-    </ul>
   </div>
-
-  <div class="my-4">
-    <div class="d-flex align-items-center mb-3">
-      <h4 class="mt-3">
-        <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt32521b6ce9c6b5ef/636c23102e673b30d507d84e/need-help-logo-color-32px.png');"></span>
-        Need help?
-      </h4>
-    </div>
-    <ul class="ul-col-md-2 ul-col-1">
-      <li>
-        <a href="en/starting-with-the-elasticsearch-platform-and-its-solutions/current/get-support-help.html">How to get support</a>
-      </li>
-      <li>
-        <a href="https://discuss.elastic.co/">Forums</a>
-      </li>
-      <li>
-        <a href="https://www.elastic.co/training/">Training</a>
-      </li>
-      <li>
-        <a href="en/starting-with-the-elasticsearch-platform-and-its-solutions/current/troubleshooting-and-faqs.html">Troubleshooting and FAQs</a>
-      </li>
-      <li>
-        <a href="en/starting-with-the-elasticsearch-platform-and-its-solutions/current/introducing-elastic-documentation.html">Introducing Elastic documentation</a>
-      </li>
-      <li>
-        <a href="https://www.elastic.co/events/?tab=1&solution=null&event=Meetup&region=null&language=English&industry=null">Meetups and virtual events</a>
-      </li>
-      <li>
-        <a href="https://ela.st/slack">Community Slack</a>
-      </li>
-    </ul>
+</div>
+
+<div class="my-4">
+  <div class="d-flex align-items-center mb-3">
+    <h3 class="mt-3">
+      <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt32521b6ce9c6b5ef/636c23102e673b30d507d84e/need-help-logo-color-32px.png');"></span>
+      Need help?
+    </h3>
   </div>
-
+  <ul class="ul-col-md-2 ul-col-1">
+    <li>
+      <a href="en/starting-with-the-elasticsearch-platform-and-its-solutions/current/get-support-help.html">How to get support</a>
+    </li>
+    <li>
+      <a href="https://discuss.elastic.co/">Forums</a>
+    </li>
+    <li>
+      <a href="https://www.elastic.co/training/">Training</a>
+    </li>
+    <li>
+      <a href="en/starting-with-the-elasticsearch-platform-and-its-solutions/current/troubleshooting-and-faqs.html">Troubleshooting and FAQs</a>
+    </li>
+    <li>
+      <a href="en/starting-with-the-elasticsearch-platform-and-its-solutions/current/introducing-elastic-documentation.html">Introducing Elastic documentation</a>
```

Comment:
I think this is important, so would consider making it somewhere in the beginning of the list.